packages/auth/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "cross-env NODE_ENV=development NODE_OPTIONS=--dns-result-order=ipv4first nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
     "start:prod": "TS_NODE_BASEURL=./build/dist/src node -r tsconfig-paths/register build/dist/src/index.js",
-    "test": "jest --coverage --silent --noStackTrace && yarn test:compilation",
+    "test": "yarn test:compilation && jest --coverage --silent --noStackTrace",
```

Comment:
I'm switching these this way whenever I see them for two reasons:
- Compile check is much faster, no point of proceeding if it fails
- I can pass additional flags to the last command `yarn test -- --watch file.ts`